node/src/components/linear_chain_sync.rs

```diff
@@ -91,7 +92,7 @@ impl<I: Clone + PartialEq + 'static> LinearChainSync<I> {
         chainspec: &Chainspec,
         storage: &Storage,
         init_hash: Option<BlockHash>,
```

Comment:
(Not necessarily in this PR.)
Should we rename that to `trusted_block_hash`, to avoid confusion with the initial state root hash?